src/components/ConnectToXeroButton/types.ts

```diff
@@ -0,0 +1,10 @@
+import type {PolicyConnectionName} from '@src/types/onyx/Policy';
+
+type ConnectToXeroButtonProps = {
```

Comment:
As I've mentioned, let's move these to common type?

Reply:
We could do this along with making same button implementation(based on integration type) for Xero and Quickbooks